README.md

```diff
@@ -31,7 +31,6 @@ PDA for ASE
     TEXT_TO_SPEECH_APIKEY=<TTS-APIKEY>
     TEXT_TO_SPEECH_IAM_APIKEY=<TTS_IAM_API-KEY>
     TEXT_TO_SPEECH_URL=<TTS-URL>
-    TEXT_TO_SPEECH_AUTH_TYPE=iam
```

Comment:
Let's keep that?

Reply:
Das wurde nirgendwo verwendet

Reply:
scheinbar nutzt das der Watson service indirekt.

    Directory provided by the environment variable IBM_CREDENTIALS_FILE
    Your system's home directory
    Your current working directory (the directory Node is executed from)
die scannt er für .env files

Reply:
Das braucht man nur, wenn sich die API selbst ihre Authentifizierungs-Daten suchen soll. Wir brauchen das nicht, weil wir den IamAuthenticator explizit übergeben:
https://github.com/JaydenUlysses/Olaf-the-coolest-of-Bots/blob/d91071ab42e5cc645d9e8d5c3b938824d9585b6b/src/connectors/speechToText/speechToTextConnector.ts#L8-L11